### ISSUE TITLE
Error when deleting current folder

### ISSUE THREAD

**Issue by cewbdex** [zhanghai/MaterialFiles#536]
I can select a folder, enter it and then delete it, should have other behaviour than an ugly error message, maybe it could visit a parent directory.
![Screenshot_2021-07-19-10-49-53-034_me zhanghai android files](https://user-images.githubusercontent.com/9021297/126157526-4904b35c-3c1a-437d-9af3-213c9466137a.jpg)
![Screenshot_2021-07-19-10-50-01-656_me zhanghai android files](https://user-images.githubusercontent.com/9021297/126157536-66a12e2f-1b8a-488e-ab9d-83a1ab72c782.jpg)

**Comment by zhanghai** [comment]
Yes this is a small known bug - the deletion should remove the deleted folder from breadcrumbs.

**Comment by cewbdex** [comment]
I can no longer select a folder and then enter it. So I guess we can close this?

**Comment by zhanghai** [comment]
I didn't change anything about this and I just tested again, I can still enter a deleted directory via breadcrumbs. So still an issue I think...
